test/ejabberd_helper.erl

```diff
@@ -20,6 +20,9 @@ stop_ejabberd() ->
 
 -spec use_config_file(any(), file:name_all()) -> ok.
 use_config_file(Config, ConfigFile) ->
+    %% To avoid "NIF library already loaded (reload disallowed since OTP 20)."
+    %% error on fast_tls restart, always keep it running
+    application:ensure_all_started(fast_tls),
```

Comment:
because it does not work without it ;)
But seriously, I think application server stops application deps OR fast_tls is started not using application server.

Be aware, we had no problems with it before OTP 20.
TLDR: there should be check in fast_tls, but there is no. So, a hack for now.